docs/input-main.md

```diff
@@ -62,7 +62,7 @@
 
     [tddft](#tddft) | [td_scf_thr](#td_scf_thr) | [td_dt](#td_dt) | [td_force_dt](#td_force_dt) | [td_vext](#td_vext) | [td_vext_dire](#td_vext_dire) | [td_timescale](#td_timescale) | [td_vexttype](#td_vexttype) | [td_vextout](#td_vextout) | [td_dipoleout](#td_dipoleout) | [ocp](#ocp) | [ocp_set](#ocp_set)
 
-- [DFT+U correction](#DFT_U-correction) (Under tests)
+- [DFT+U correction](#DFT+U-correction) (Under tests)
```

Comment:
This change makes it a dead link for me😂

Reply:
> 

The name of the section is DFT+U correction. How is it possible that yours work?